src/App/App.tsx

```diff
@@ -48,12 +48,12 @@ export default function App() {
             element={<BankingApplications />}
           />
           <Route
-            path={appRoutes.banking_applications + "/:id"}
+            path={`${appRoutes.banking_applications}/:id`}
             element={<BankingApplication />}
           />
           <Route path={appRoutes.applications} element={<Applications />} />
           <Route
-            path={appRoutes.applications + "/:id"}
+            path={`${appRoutes.applications}/:id`}
```

Comment:
some lint rules are just style related that we  can safely stick to our existing code. I tried biome last night with less invasive rules #2666 , that we can make stricter incrementally prioritizing performance, then correctness, then lastly style related

another point to consider is switching to biome formatter is best with the biome vscode extension as replacement for `prettier` - so if we're going to adopt this on web-app, we should also adopt on devops, devops-ses at once so we don't need to switch from prettier - biome from time to time